arbitrum-docs/run-arbitrum-node/data-availability-committees/02-deploy-das.md

```diff
@@ -273,11 +273,11 @@ Archive DA servers are servers that don't discard any data after expiring. Each
 
 To activate the "archive mode" in your DAS, set the parameter `discard-after-timeout` to `false` in your storage method. For example:
 
-```bash
+```shell
 --data-availability.s3-storage.discard-after-timeout=false
 ```
 
-Note that `local-file-storage` doesn't discard data after expiring, so the option `discard-after-timeout` is not available.
+Note that `local-file-storage` doesn't discard data after expiring by default, but expiration can be enabled with `enable-expiry` (the option `discard-after-timeout` is not available for this storage option).
```

Comment:
I think you don't need to mention discard-after-timeout here. I'll probably rename that option on s3 storage at some point, or have both options. enable-expiry is a better name

Reply:
Alright, removed the mention to discard-after-timeout. Thanks!